keps/sig-auth/2718-20210511-client-exec-proxy/kep.yaml

```diff
@@ -0,0 +1,54 @@
+title: KEP Template
+kep-number: 2718
+authors:
+  - "@nckturner"
+  - "@micahhausler"
+owning-sig: sig-auth
+participating-sigs:
+  - sig-cli
+  - sig-api-machinery
+status: provisional
+creation-date: 2021-05-11
+reviewers:
+  - "@mikedanese"
+  - "@enj"
+  - "@liggitt"
+approvers:
+  - "@mikedanese"
+  - "@enj"
+  - "@liggitt"
+prr-approvers:
+  - "@deads2k"
```

Comment:
Sure.  Also, please create a file like https://github.com/kubernetes/enhancements/blob/master/keps/prod-readiness/sig-auth/1393.yaml when the overall design is worked out.